custom_image_utils/shell_script_generator.py

```diff
@@ -35,32 +35,51 @@
   local -r cmd="$*"
 
   for ((i = 0; i < 3; i++)); do
-    if eval "$cmd"; then return 0 ; fi
+    time eval "$cmd" > "/tmp/{run_id}/install.log" 2>&1 && retval=$? || {{ retval=$? ; cat "/tmp/{run_id}/install.log" ; }}
```

Comment:
Why was this change made?

Reply:
to reduce noise in the log from successful runs of retried tasks, and to track the duration of long-running tasks.

This will only print the log on error